lookout/style/format/virtual_node.py

```diff
@@ -144,3 +142,6 @@ def from_node(node: bblfsh.Node, file: str, path: str,
                                        line=node.end_position.line,
                                        col=node.end_position.col),
                               path=path)
+
+
+AnyNode = Union[VirtualNode, Node]
```

Comment:
This is my concern... Interfaces of those two are very different, and our type hints are not always consistent.

Each Node used to be backed by a VirtualNode before. Is it possible to always feed VirtualNode-s and take `VirtualNode.node` attribute?

Reply:
parents never used to be backed by VirtualNode-s, that's why we have this union in features. I'll add a restriction on types for each feature to clarify everything